src/mlpack/core/tree/binary_space_tree/binary_space_tree.hpp

```diff
@@ -390,9 +390,6 @@ class BinarySpaceTree
   //! Return the minimum distance from the center of the node to any bound edge.
   ElemType MinimumBoundDistance() const;
 
-  //! Modify the minimum distance from the center of the node to any bound edge.
-  ElemType& MinimumBoundDistance() { return minimumBoundDistance; }
```

Comment:
Right, it looks like the other version of this overload calls `bound.MinWidth()`.  So the `minimumBoundDistance` member appears to be entirely unused in `BinarySpaceTree` and we can remove it.

Reply:
And this sloves the failing tests issue as well :)